src/com/lilithsthrone/game/dialogue/story/CharacterCreation.java

```diff
@@ -1797,9 +1797,19 @@ public void effects() {
 						Main.game.getPlayer().setMoney(5000);
 						Main.game.getPlayer().equipMainWeaponFromNowhere(AbstractWeaponType.generateWeapon(WeaponType.MELEE_CHAOS_RARE, DamageType.FIRE));
 						
-						AbstractItem spellBook = AbstractItemType.generateItem(ItemType.getSpellBookType(Spell.ICE_SHARD));
+						if (Main.game.getPlayer().getBirthMonth().getValue() == 1||Main.game.getPlayer().getBirthMonth().getValue() == 5||Main.game.getPlayer().getBirthMonth().getValue() == 9) {
```

Comment:
Suggestion for readability (at least, imo):

```suggestion
						if (Main.game.getPlayer().getBirthMonth().getValue() % 4 == 1) {
```

And then `% 4 == 2`, `% 4 == 3` and `% 4 == 0` for the others.

Reply:
+1